docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,57 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _frequently_asked_questions:
+
+##########################
+Frequently Asked Questions
+##########################
+
+#. I just purchased my motor, how do I get started with Vertiq's IQ Control Center software? How can I communicate with my module?
+    * Issues with spinning your module can be related to improper configuration of the motor. Control Center is the recommended tool for configuring your module. In order to connect with Control Center, refer to your module’s family page found under the sidebar's Modules section. Here, you will find the hardware setup instructions for your specific module. After your hardware is set up, refer to our :ref:`Control Center Manual <control_center_start_guide>` for instructions on how to install and run Control Center. Then, follow the instructions on configuring your module using Control Center depending on its firmware: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`. Please see our documentation under the Tutorials section of the sidebar for additional information, such as integrating with your flight controller, retrieving flight controller logs, and debugging communication issues. 
```

Comment:
I think the "In order to connect with Control Center, refer to your module's family page found..." is a bit misleading

They need to set up the hardware in order to connect to control center, and they need the family page to learn about the hardware setup.